modules/core/karma.conf.js

```diff
@@ -10,30 +13,44 @@ module.exports = function(config) {
 
     // frameworks to use
     // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-    frameworks: ['mocha'],
+    frameworks: ['jasmine', 'karma-typescript'],
 
 
     // list of files / patterns to load in the browser
     files: [
-      'browser/tests.js'
+      'dist/browser/BitGoJS.min.js',
+      'test/browser/karmaHelper.js',
+      { pattern: 'test/browser/**/*.ts' }
     ],
 
-    client: {
-      mocha: {
-        timeout: '10000'
-      }
-    },
-
-    // list of files to exclude
+    plugins: [
+      require('karma-jasmine'),
+      require('karma-typescript'),
+      require('karma-chrome-launcher'),
+    ],
+    // list of files / patterns to exclude
     exclude: [
+      'node_modules',
     ],
 
 
     // preprocess matching files before serving them to the browser
     // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
     preprocessors: {
+      'test/browser/**/*.ts': ['karma-typescript'] ,
     },
-
+    karmaTypescriptConfig: {
+      bundlerOptions: {
+          entrypoints: /\.spec\.ts$/,
+      },
+      compilerOptions: {
+          sourceMap: true,
+          target: 'ES5',
```

Comment:
question - shouldn't we target what we're supporting? node8+? otherwise we get some weird divergence in JS/browser/node versions in tests

Reply:
eh, for these browser tests it doesn't really matter since we're only using these compilerOptions to standup the karma test. But sure, i went ahead & changed to use the same target & libs we use in microservices (es6).

Reply:
awesome